CefSharp.OutOfProcess.Interface/IOutOfProcessClientRpc.cs

```diff
@@ -35,8 +36,22 @@ public interface IOutOfProcessClientRpc
         /// <param name="parentHwnd">parent Hwnd</param>
         /// <param name="url">start url</param>
         /// <param name="browserId">browser id</param>
+        /// <param name="requestContextPreferences">Request Context Preferences to set.</param>
         /// <returns>Task</returns>
-        Task CreateBrowser(IntPtr parentHwnd, string url, int browserId);
+        Task CreateBrowser(IntPtr parentHwnd, string url, int browserId, IDictionary<string, object> requestContextPreferences);
+
+        /// <summary>
+        /// Modify RequestContext-Preferences for an existing browser.
+        /// </summary>
+        /// <param name="browserId">browser id</param>
+        /// <param name="requestContextPreferences">Request Context Preferences to set.</param>
+        void SetRequestContextPreferences(int browserId, IDictionary<string, object> requestContextPreferences);
```

Comment:
How often do you need to set multiple preferences at once? If I'm honest I don't love this. There's no error reporting, absolutely no idea if the preferences were set.

Personally I'd prefer to see something like the following where you get the bool and error string returned from the `SetPreference` call.

```c#
Task<SetRequestContextPreferenceResponse> SetRequestContextPreferenceAsync(int browserId, string name, object value);
```